### PR TITLE
Renamed the installing deploy folders and section to getting started

### PR DESCRIPTION
**Pull request by jonat123**

### CODE REVIEW

**Comment by sofietoft** [comment]
Looks good 💪 
Remember version 10 as well, and also redirects for the articles!

**Comment by jonat123** [comment]
@sofietoft I've moved it to the v10 section now as well and I have added redirects, could you please have a look again just to sanity check whether i have done everything correct? :D

File: 10/umbraco-deploy/.gitbook.yaml [hunk 1]
```diff
@@ -4,3 +4,11 @@ root: ./
     readme: README.md
     summary: SUMMARY.md
 
+installing-deploy/installing-deploy: getting-started/install-configure-md
+installing-deploy/install-configure: getting-started-install-configure.md
+installing-deploy/install-configure/existing-site: getting-started/install-configure.md
+installing-deploy/install-configure/new-site: getting-started/install-configure.md
```
Comment (sofietoft) [inline 10/umbraco-deploy/.gitbook.yaml:10]: You might be able to add the direct links here, if we want want. So, link to the specific expandable, which should also automatically open the expandable, I think 🤔 

Example: `getting-started/install-configure#new-site`

File: 10/umbraco-deploy/.gitbook.yaml [hunk 2]
```diff
@@ -4,3 +4,11 @@ root: ./
...
+installing-deploy/cicd-pipeline: getting-started/README.md
```
Comment (sofietoft) [inline 10/umbraco-deploy/.gitbook.yaml:11]: Shouldn't this be `getting-started/cicd-pipeline/README.md` ? 🤔

**Review by sofietoft — COMMENTED** [review]
Looks good 💪 
I've left a couple of comments - only on 10, but it should be applied for 11 as well.'

**Review by sofietoft — APPROVED** [review]

**Comment by jonat123** [comment]
merging in!

**Pull request merged**